### PR TITLE
Note to create a service if you extend ExceptionController

### PR DESCRIPTION
**Pull request by pamuche**
I had some trouble to get the custom exception handling with inheritance of ExceptionController working, because I hadn't a clue to make a service out of the ExceptionController of the TwigBundle.

### CODE REVIEW

File: cookbook/controller/error_pages.rst [hunk 1]
```diff
@@ -251,7 +251,25 @@ will be passed two parameters:
 Instead of creating a new exception controller from scratch you can, of course,
 also extend the default :class:`Symfony\\Bundle\\TwigBundle\\Controller\\ExceptionController`.
 In that case, you might want to override one or both of the ``showAction()`` and
-``findTemplate()`` methods. The latter one locates the template to be used.
+``findTemplate()`` methods. The latter one locates the template to be used. 
+
+.. note::
+  
+  In case of extending the :class:`Symfony\\Bundle\\TwigBundle\\Controller\\ExceptionController` 
+  you may configure a service to pass the twig-Environment and the debug-Flag to the constructor.
```
Comment (javiereguiluz) [inline cookbook/controller/error_pages.rst]: I would replace this:

```
twig-Environment and the debug-Flag
```

by this:

```
Twig environment and the ``debug`` flag
```

File: cookbook/controller/error_pages.rst [hunk 2]
```diff
@@ -251,7 +251,25 @@ will be passed two parameters:
...
+  In case of extending the :class:`Symfony\\Bundle\\TwigBundle\\Controller\\ExceptionController` 
+  you may configure a service to pass the twig-Environment and the debug-Flag to the constructor. 
+  
+  .. configuration-block::
+  
+      .. code-block:: yaml
+  
+          # app/config/config.yml
+          services:
+            my_custom_exception_controller_service:
```
Comment (javiereguiluz) [inline cookbook/controller/error_pages.rst]: In modern Symfony apps, we recommend to use shorter service names. What about replacing `my_custom_exception_controller_service` by `app.exception_controller` ?

**Comment by javiereguiluz** [comment]
@pamuche thanks for your contribution. We love comments from people who followed the docs and faced problems or didn't know how to continue. I've left some comments when reviewing your changes, but overall they look OK to me. Thanks!

**Comment by WouterJ** [comment]
Thank you @pamuche for creating this PR and your quick response to the review!

I've merged your pull request into the 2.3 version of the docs (that's the oldest version we maintain) and will merge it up to master from there. After the merge, I've fixed indentation (4 spaces) and added the XML and PHP format for the service in 3b0ea60. Feel free to comment if you have any more questions.

**Pull request closed without merging**